src/main/java/uk/gov/hmcts/darts/event/controller/EventsController.java

```diff
@@ -64,6 +67,9 @@ public class EventsController implements EventApi {
     private final DataAnonymisationService dataAnonymisationService;
     private final UserIdentity userIdentity;
 
+    @Value("${darts.event-obfuscation.enabled}")
+    private final boolean eveventObfuscationEnabled;
```

Comment:
Typo, should be `eventObfuscationEnabled`?

Reply:
Updated :)